### ISSUE TITLE
training loss

### ISSUE THREAD

**Issue by hdmjdp** [wangtianrui/DCCRN#15]
can you give me the loss log? thx

below is my loss 

Epoch 0: 100%|██████████| 21209/21209 [15:24:41<00:00,  2.62s/it, loss=13.395, v_num=0]ts/DCCRN# tail -f nohup.out
Epoch 00000: val_loss reached 13.25967 (best 13.25967), saving model to /data/juicefs_speech_tts_v3/11139034/work/ptts/DCCRN/logs/_ckpt_epoch_0.ckpt as top 5
Epoch 1:  29%|██▉       | 6162/21209 [4:45:09<11:36:18,  2.78s/it, loss=14.446, v_num=0, val_loss=13.3]

is it right?
